src/accessibility/context_menu/StyledMenuItemCheckbox.tsx

```diff
@@ -18,37 +18,53 @@ limitations under the License.
 
 import React from "react";
 
-import { Key } from "../../Keyboard";
 import { useRovingTabIndex } from "../RovingTabIndex";
 import StyledCheckbox from "../../components/views/elements/StyledCheckbox";
+import { KeyBindingAction } from "../KeyboardShortcuts";
+import { getKeyBindingsManager } from "../../KeyBindingsManager";
 
 interface IProps extends React.ComponentProps<typeof StyledCheckbox> {
     label?: string;
     onChange(); // we handle keyup/down ourselves so lose the ChangeEvent
-    onClose(): void; // gets called after onChange on Key.ENTER
+    onClose(): void; // gets called after onChange on KeyBindingAction.ActivateSelectedButton
 }
 
 // Semantic component for representing a styled role=menuitemcheckbox
 export const StyledMenuItemCheckbox: React.FC<IProps> = ({ children, label, onChange, onClose, ...props }) => {
     const [onFocus, isActive, ref] = useRovingTabIndex<HTMLInputElement>();
 
     const onKeyDown = (e: React.KeyboardEvent) => {
-        if (e.key === Key.ENTER || e.key === Key.SPACE) {
+        let handled = true;
+        const action = getKeyBindingsManager().getAccessibilityAction(e);
+
+        switch (action) {
+            case KeyBindingAction.Space:
+                onChange();
+                break;
+            case KeyBindingAction.Enter:
+                onChange();
+                // Implements https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-12
+                onClose();
+                break;
```

Comment:
```suggestion
            case KeyBindingAction.Space:
                onChange();
                // fallthrough
            case KeyBindingAction.Enter:
                // Implements https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-12
                onClose();
                break;
```

Reply:
If enter is pressed, then `onChange()` won't get called, right?